packages/datetime/src/dateRangePicker.tsx

```diff
@@ -223,7 +223,7 @@ export class DateRangePicker extends AbstractPureComponent2<IDateRangePickerProp
             hoverValue: [null, null],
             leftView,
             rightView,
-            selectedShortcutIndex: this.props.selectedShortcutIndex || -1,
+            selectedShortcutIndex: this.props.selectedShortcutIndex !== undefined ? this.props.selectedShortcutIndex : -1,
```

Comment:
noticed this existing bug where if you selected the first shortcut, it would not get highlighted. Explicit checks are better and in the dev guidelines

Reply:
good catch... I think this fixes https://github.com/palantir/blueprint/issues/3807